lib/Basics/FileUtils.cpp

```diff
@@ -756,17 +756,17 @@ void makePathAbsolute(std::string& path) {
   }
 }
 
-std::string slurpProgram(std::string const& program) {
+namespace {
+
+std::string slurpProgramInternal(std::string const& program,
+                                 std::vector<std::string>& moreArgs) {
```

Comment:
That "moreArgs" is passed by mutable reference makes this API look a bit weird. Why not use a const reference for it?
```suggestion
                                 std::vector<std::string> const& moreArgs) {
```